internal/kafka/internal/migrations/20221104090000_remove_accepted_cluster_that_have_not_yet_been_created_in_cluster_provider.go

```diff
@@ -0,0 +1,23 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+func removeAcceptedClusterThatHaveNotYetBeenCreatedInClusterProvider() *gormigrate.Migration {
+	return &gormigrate.Migration{
+		ID: "20221104090000",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.Exec("DELETE FROM clusters where status='cluster_accepted' AND (cluster_id='' OR cluster_id IS NULL);").Error; err != nil {
```

Comment:
As discussed through other channels it would probably be best to do it based on ID as this is too broad even if only executed once.

Reply:
A possible alternative is to ask SRE to delete that cluster to avoid needing to add the migration